<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraSettingsDialog</class>
 <widget class="QDialog" name="CameraSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Exposure time</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="dspnExposureTime">
     <property name="suffix">
      <string>ms</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>ROI x</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>ROI y</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>ROI width</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>ROI height</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="spnROIX"/>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="spnROIY"/>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="spnROIWidth"/>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="spnROIHeight"/>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="buttSaveClose">
     <property name="text">
      <string>Save &amp;&amp; close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
